lambdas/test-lambda/src/main/java/uk/gov/di/data/lep/GroDeathNotificationMinimisation.java

```diff
@@ -0,0 +1,88 @@
+package uk.gov.di.data.lep;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import uk.gov.di.data.lep.classes.GroDeathEventDetails;
+import uk.gov.di.data.lep.classes.GroDeathEventNotification;
+import uk.gov.di.data.lep.classes.GroDeathEventEnrichedData;
+import uk.gov.di.data.lep.enums.EnrichmentField;
+import uk.gov.di.data.lep.enums.EventType;
+
+import java.util.Arrays;
+import java.util.UUID;
+
+public class GroDeathNotificationMinimisation implements RequestHandler<GroDeathEventEnrichedData, GroDeathEventNotification> {
+    @Override
+    public GroDeathEventNotification handleRequest(GroDeathEventEnrichedData enrichedData, Context context) {
+
+
+        // Set up general (~meta) output data
+        GroDeathEventNotification eventNotification = new GroDeathEventNotification();
+        eventNotification.eventId = UUID.randomUUID();
+        eventNotification.eventType = EventType.DEATH_NOTIFICATION;
+        eventNotification.sourceId = enrichedData.sourceId;
+        eventNotification.dataIncluded = true;
+        eventNotification.enrichmentFields = Config.enrichmentFields;
+
+        var eventDetails = new GroDeathEventDetails();
+
+        for (var enrichmentField : Config.enrichmentFields) {
```

Comment:
instead of a for loop where we loop through all checks everytime, could we do some checks of if the field exists in the list